lib/ap/amd64/sigsetjmp.s

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file is part of the UCB release of Plan 9. It is subject to the license
+ * terms in the LICENSE file found in the top-level directory of this
+ * distribution and at http://akaros.cs.berkeley.edu/files/Plan9License. No
+ * part of the UCB release of Plan 9, including this file, may be copied,
+ * modified, propagated, or distributed except according to the terms contained
+ * in the LICENSE file.
+ */
+ 
+.globl sigsetjmp
```

Comment:
no comment at all about what this is doing? Why the andl? the whole thing makes no sense to me.